### ISSUE TITLE
[ENHANCEMENT] maintain the packages on OBS

### ISSUE THREAD

**Issue by Vicente-Cheng** [harvester/harvester#3587]
**Is your enhancement related to a problem? Please describe.**
We have some packages installed from luet, but most of them need to be updated.
We should maintain our package list on our OBS. 

**Describe the solution you'd like**
Maintain packages ourselves on OBS.

**Describe alternatives you've considered**

**Additional context**
For the short term, we can download the newer version directly.

**Comment by bk201** [comment]
- [x] virtctl (prefer to use the same version with kubevirt images)
- [x] yip

**Comment by Vicente-Cheng** [comment]
Now we maintain `kubevirt` (for `virtctl`) and yip ourselves on https://build.opensuse.org/project/show/isv:Rancher:Harvester:ExtraPackages:SLE-15-SP4

base OS is updated:
https://build.opensuse.org/package/show/isv:Rancher:Harvester:BaseOS53/baseos

- [x] Test with harvester-installer by latest base OS

**Comment by Vicente-Cheng** [comment]
Build the harvester ISO with local base os.
virtctl version:
```
harvester-node-0:~ # virtctl version
Client Version: version.Info{GitVersion:"v0.54.0", GitCommit:"v0.54.0", GitTreeState:"clean", BuildDate:"2022-06-01T12:08:36Z", GoVersion:"go1.19.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{GitVersion:"v0.54.0", GitCommit:"v0.54.0", GitTreeState:"clean", BuildDate:"2022-06-01T12:08:36Z", GoVersion:"go1.17.13", Compiler:"gc", Platform:"linux/amd64"}
```

yip version:
```
harvester-node-0:~ # rpm -qa |grep yip
yip-v1.0.0-150400.1.1.x86_64
```

Also, verify with interactive installation and PXE installation.
Basic operations on the harvester look good.

I will cut the base os for this issue.

**Comment by albinsun** [comment]
Tested in `v1.2-4d7b1196-head`

## Test Environment
* Harvester
  * version: `v1.2-4d7b1196-head`
  * profile: qemu/KVM, single node
  * **ui-source**: Auto

## Verification
### Version
1. virtctl $~~~~$ $\color{green}{\textsf{PASSED}}$
   * OBS: `0.54.0-150400.3.18.10.x86_64` [(Ref.)](https://build.opensuse.org/package/live_build_log/isv:Rancher:Harvester:ExtraPackages:SLE-15-SP4/kubevirt/standard/x86_64)
   * v1.2-head
   ```
   harvester-node-0:~ # rpm -qa | grep virtctl
   kubevirt-virtctl-0.54.0-150400.3.18.10.x86_64
   ```
   * Major version is the same as system's (**0.54.0-150400**.3.16.1) 
   ```
   harvester-node-0:~ # kubectl get deployments -A -o jsonpath='{range .items[*]}{@.metadata.name}{" "}{@.spec.template.spec.containers[*].image}{"\n"}{end}' | grep virt | column -t
   virt-api         registry.suse.com/suse/sles/15.4/virt-api:0.54.0-150400.3.16.1
   virt-controller  registry.suse.com/suse/sles/15.4/virt-controller:0.54.0-150400.3.16.1
   virt-operator    registry.suse.com/suse/sles/15.4/virt-operator:0.54.0-150400.3.16.1
   ```

2. yip $~~~~$ $\color{green}{\textsf{PASSED}}$
   * OBS: `v1.0.0-150400.1.1.x86_64` [(Ref.)](https://build.opensuse.org/package/live_build_log/isv:Rancher:Harvester:ExtraPackages:SLE-15-SP4/yip/standard/x86_64)
   * v1.2-head
   ```
   harvester-node-0:~ # rpm -qa | grep yip
   yip-v1.0.0-150400.1.1.x86_64
   ```

### Basic VM Operations
- [x] Create
- [x] Restart VM
- [x] Soft reboot
- [x] Stop-Start
- [x] Pause-Unpause
- [x] Edit
- [x] Add volume
- [x] Delete

## Note
1. OS Version
   ```
   harvester-node-0:~ # cat /etc/os-release
   NAME="SLE Micro"
   VERSION="5.3"
   VERSION_ID="5.3"
   PRETTY_NAME="Harvester v1.2"
   ID="sle-micro-rancher"
   ID_LIKE="suse"
   ANSI_COLOR="0;32"
   CPE_NAME="cpe:/o:suse:sle-micro-rancher:5.3"
   VARIANT="Harvester"
   VARIANT_ID="Harvester-v1.2-20230629"
   GRUB_ENTRY_NAME="Harvester v1.2"

**Comment by albinsun** [comment]
Close as testted in `v1.2-4d7b1196-head`, fix for v1.1 backport will be tracked on harvester#3587
[[ENHANCEMENT] maintain the packages on OBS](https://github.com/harvester/harvester/issues/3587).
